pkg/security/probe/model_windows.go

```diff
@@ -21,6 +21,11 @@ func NewWindowsModel(_ *WindowsProbe) *model.Model {
 			// TODO(safchain) remove this check when multiple model per platform will be supported in the SECL package
 			if !strings.HasPrefix(field, "exec.") &&
 				!strings.HasPrefix(field, "exit.") &&
+				!strings.HasPrefix(field, "create_file.") &&
+				!strings.HasPrefix(field, "create_registry_key.") &&
```

Comment:
I would rather use a registry event type. So field would be "registry.create_key', 'registry.open_key', etc.